src/stories/Library/nav-grid/nav-grid.scss

```
@@ -0,0 +1,74 @@
+.nav-grid {
+  @include layout-container($layout__max-width--large, 0);
+  @include vertical-spacing();
+}
+
+.nav-grid--simple {
+  .nav-teaser__subtitle {
+    display: none;
+  }
+}
```

```suggestion
.nav-grid--simple.nav-teaser__subtitle {
   display: none;
  }
```

i guess no reason to nest

---

the reason to nest is to make it easier to read :)